build/index.js

```diff
@@ -18,7 +18,7 @@ exports.onBeforeBuild = function (api, app, config, cb) {
              'var js, fjs = d.getElementsByTagName(s)[0];',
              'if (d.getElementById(id)) {return;}',
              'js = d.createElement(s); js.id = id;',
-             'js.src = "//connect.facebook.net/en_US/all.js";',
+             'js.src = "//connect.facebook.net/en_US/sdk.js";',
```

Comment:
Does this still work in the simulator? I seem to recall some issues when not using `all`.